src/ui/settings/index.js

```diff
@@ -60,6 +58,18 @@ export default function setupSettings(kbnServer, server, config) {
   function remove(key) {
     return set(key, null);
   }
+
+  function mirrorEsStatus() {
+    const esStatus = kbnServer.status.getForPluginId('elasticsearch');
+
+    copyStatus();
+    esStatus.on('change', copyStatus);
+
+    function copyStatus() {
+      const { state } = esStatus;
+      status[state](state === 'green' ? 'Ready' : `Elasticsearch plugin is ${state}`);
```

Comment:
I feel like this line should change into a proper conditional, or maybe just extract the ternary into a `const` variable.